RELEASE_NOTES.md

```diff
@@ -1,15 +1,13 @@
 # Upcoming
 
-- fix: Compiled lambdas now close only on non-ghost variables (https://github.com/dafny-lang/dafny/pull/2854)
-- fix: Crash in the LSP in some code that does not parse (https://github.com/dafny-lang/dafny/pull/2833)
+See [docs/dev/news/](docs/dev/news/).
```

Comment:
I'd add a stronger comment that this file is no longer used as part of the release process.

Reply:
This is a user-facing file, though, so should we add this type of comments there?
Maybe as a comment that doesn't get rendered?

Reply:
I don't think we should add a comment about the release process here, but I like the idea of also redirecting users to the folder where the fragments are so that they can see what to expect in the next release.

Reply:
Ah I missed the fact that the script is still updating this file, I thought it was going to be frozen in this state forever more. Nevermind. :)